src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/memberdays/MemberDaysReport.java

```diff
@@ -178,10 +178,14 @@ private String getHearingDuration(DateListedTypeItem dateListedTypeItem) {
         long breakResumeDuration = 0;
         if (dateListedTypeItem.getValue().getHearingTimingBreak() != null
             && dateListedTypeItem.getValue().getHearingTimingResume() != null) {
-            var hearingBreak = LocalDateTime.parse(dateListedTypeItem.getValue().getHearingTimingBreak(),
-                OLD_DATE_TIME_PATTERN3);
-            var hearingResume = LocalDateTime.parse(dateListedTypeItem.getValue().getHearingTimingResume(),
-                OLD_DATE_TIME_PATTERN3);
+            var hearingBreakStart = dateListedTypeItem.getValue().getHearingTimingBreak();
```

Comment:
There's a lot of code duplication in converting the string to a date. Could you refactor this by extracting out the common code into a method?

Reply:
duplication removed.